protocol/src/test/java/org/fido/iot/protocol/To2EcRsaTest.java

```diff
@@ -0,0 +1,384 @@
+      + "oqOmKq1r3qqiN2y+o5Yw+gjK0DGmtBbWEhATebb8ao8EvjHE0+EGl3rcn8Ruzzur\n"
+      + "vm0behy+j16cW/w1RhnB1t18aQJ86/RF2kYMClBw8hk72t2pI5sAf6OZX8mUVXhN\n"
+      + "b6ZCdcV3Qc1lbUNm6PEBrIag9u6CMJ84116nUoguEqkVGjNjrbMVAYEAfdWkY/C1\n"
+      + "s4c/SYwfHgHiPdlzAgMBAAECggGBAIjqGIWxXq3Wp6monx0YbUylZBvj8UrT1i4j\n"
+      + "2EocGRncAlbcARVLkLx9iH3g28g3DE5fpEljKXOf14iUU82zMcEy+PHKOCwIthsP\n"
+      + "SquVMLnhZplhP6TzXDoqzmc1YbeyKgToHxG8B7cBtaX9mzASbwoLKSYMem1k3eWn\n"
+      + "XZxe/9zuZXhb0nc4pI6pY1LqWLT82bMSRD0JCxC0wYXI03FSY6Q4WFR0r5L7HG2h\n"
+      + "gGs4n6M4jiVyxExAh0WQETe2l+DTKV5SDWIqCcUqvu07XJjyh0IYHG6rLwaYFLp1\n"
+      + "IExDQ09cvxnTJPlBwq6waeO6WQVnaCJZBAvXpb/0zGzDlq4kto3tdzDjns98XpD9\n"
+      + "+Ty9MrwmNqAka4zHGfPdw194Tng7k5Wia2ieXQMUdBsNiupUq1gKdtWjn7t1tEfA\n"
+      + "nRTUCdZ2ypZsakBnkVK7DBrM7wAfIcbC93+FKgUzNO18e29libj3EssXTrKZMJtn\n"
+      + "FxvxOivRPEEXj/B/6JXTKlAWq1PhEQKBwQDeIXMCLvUouvBniuUtDi3WFc7EvzMX\n"
+      + "IDf1p0njjURAphLOZalkVdAwDW1qufuJzWQItqfDvcW5m+ExOzDF+ckdiymKDbSo\n"
+      + "i4UZAxK3dcuF6925WkL3BPPIrFSy2LKhrt+KdFm36M30hojtUTTfwmlXTvdgmSnt\n"
+      + "VOoGXtiqV6soHbCb6Ky+iJXeE6qHjTdDbX71AqmcmqHPqbGUi3Eq1XqtoptiXXwq\n"
+      + "uLeFiPhZXYYs5psuhIZEE8hiwo7PznuNx6UCgcEAxmrTlKVx2Y3T4+KByea6Ko6A\n"
+      + "2mYcAJVTtk5s8OX7aR+/wXUJMEMbxnv/Gb76W+zWRNmNIy1koxVNkuP1m80FS48W\n"
+      + "nt4waDGKjTZ+dPAEUEFzyfnpWQXTDkkrwjxrxfuAxnPNCU4cIZ9fQ4sTnKs8tFU6\n"
+      + "k8bzyG6lVEiUJN1H9z5UJKViXYGD8ySd7WsIccZqEgcWykHyqly5vH+4VGa08Bq2\n"
+      + "IeDXsNU+GOZTJQmZ55DfuI1qEtykqZD3vpxDjhE3AoHANceNa7rouirAZ6E2KTlk\n"
+      + "mY+AFxxfhzrSsZQGDKS4IJrUtMniQBuSov+tPQi4QL3MeVuXtIlupKh71gqJnXUD\n"
+      + "XzbBIN5+ncvhtpOK5YYMd2kKroSO7/Vv/GbUvtQ+M2z+TCwtSJyBuLEV17cwgPg1\n"
+      + "vuCsfeO3RfGoE1DFTkCZfMXra2Pi5uJ0vmaug9EJzzZkZXd2jybVBE0GZHt6M6+p\n"
+      + "2v4idqdBl6pPLHJkBmbhRF+0tId7ZmIPwvFt2Bwax9HNAoHBALB5OP/fengltczc\n"
+      + "m9UoWnIZVq9o3AUEy3S9IhXMZySjsVhuMKL1PRH3HPgYLPJvnk+UMvyTlguamelI\n"
+      + "yR0uZ0RB2ruoIOO7FGDuk+CLzTpYTJTaqtmb3ZC6MapLNQvc2jcnrbhV+Z1J81/O\n"
+      + "pb3Va2N03mmrLQ4aov9ooiJ7kVNs40fItMk3tVUfPzt31AXllFe+aF2x9BPj7uqh\n"
+      + "qh+fTdSz3UQah5xmvRlGWbL6bCFScDjjxfFKv0V1g3PL5iXIVwKBwQC+RXLGGf/B\n"
+      + "ghj5FEsDx/6fu/f0FIxMA/hWfcWML219Mx3vnC2t91rtmKfhy9LnHl64XCuf64ly\n"
+      + "iAPVgWCbnOyFG3+DMlVgIeYMvanfuyRF7C4bUxQUjEwq39D3pgU46bMrxskcM1HI\n"
+      + "VMr2UWO3BXXCNj/1WfG30t5u2rgDqa6BPHuSkfhBTxLzG5BZ6j3YZZcWQbiigqYT\n"
+      + "NAfEFNqxk2pNIMh+ryxcMLMTb1pbFlFJnaqEcqWaJMCRv3R5kcBd7nE=\n"
+      + "-----END RSA PRIVATE KEY-----\n";
+  Composite toOwnerInfo;
+  Composite toDeviceInfo;
+  Composite fromOwnerInfo;
+  Composite fromDeviceInfo;
+  private final String serverToken = guid.toString();
+  private String clientToken;
+  private byte[] storedNonce6;
+  private byte[] storedNonce7;
+  private Composite storedOwnerState;
+  private String storedCipherName;
+  private UUID storedGuid;
+
+  @BeforeAll
+  static void beforeAll() {
+    Security.addProvider(CryptoService.BCPROV);
```

Comment:
Not sure if we need to add the provider manually here, since the Cipher instance in CryptoService is directly passing the BC provider instance